backend/lunchmenuloader/homeOfficeLoader.coffee

```diff
@@ -0,0 +1,44 @@
+
+LunchmenuLoader = require './lunchmenuLoader'
+
+
+class HomeOfficeLoader extends LunchmenuLoader
+    constructor: () ->
+        @name = 'Home Office'
+        @homepage = 'http://homeofficebistro.cz/'
+        @downloadUrl = 'http://homeofficebistro.cz/tydenni-menu/'
+        @phoneNumber = '+420 778 767 768'
+        @address =
+            street: 'Radlická 3185/1C'
+            city: 'Praha 5'
+            zip: 15000
+            map:
+                lat: 50.0716481
+                lng: 14.4011792
+
+    parse: (restaurant, $) ->
+        days = ['Pondělí', 'Úterý', 'Středa', 'Čtvrtek', 'Pátek', 'Sobota', 'Neděle']
+        date = new Date()
+        day_string = days[(date.getDay() - 1)] + ' ' + date.getDate() + '/' + (date.getMonth() + 1)
```

Comment:
Co na tohle použít moment? Už je to i nakonfigurované. Progrepni si moment v loaderech. :-)